runtime/parachains/src/configuration.rs

```diff
@@ -947,6 +975,49 @@ pub mod pallet {
 			});
 			Ok(())
 		}
+
+		/// Enable or disable PVF pre-checking. Consult the field documentation prior executing.
+		#[pallet::weight((
+			T::WeightInfo::set_config_with_u32(), // TODO: should we change that to bool?
```

Comment:
This should be fine, the code for setting a u32 vs setting a boolean is pretty much identical, aside from the fact that they're different data types... perhaps we could rename the weighing function into something else to make it more clear? I just don't really have a good name, uhh maybe `set_config_with_primitive_type`?

Reply:
Hm, looking at the `trait WeightInfo` I see that there are several items that could have the name `set_config_with_primitive_type`. E.g. I don't think that there is a really big difference between `with_option_u32` and `with_u32`. I do not see a `weights.rs` file, it would be interesting to see the actual numbers there.

Right now I just removed this todo and added a comment that it should be fine. I think this could be addressed in a follow-up. We can continue the discussion here though.